src/DIRAC/FrameworkSystem/Client/BundleDeliveryClient.py

```diff
@@ -87,25 +93,25 @@ def syncDir(self, bundleID, dirToSyncTo):
                     self.log.error(f"{getpass.getuser()} does not have the permissions to update {dirToSyncTo}")
                     return S_ERROR(f"{getpass.getuser()} does not have the permissions to update {dirToSyncTo}")
         else:
-            self.log.info("Creating dir %s" % dirToSyncTo)
+            self.log.info("Creating dir", dirToSyncTo)
```

Comment:
Very very minor and cosmetic, but why not writing `directory` instead of just `dir` here? 
```suggestion
            self.log.info("Creating directory", dirToSyncTo)
```

Same thing line `107`, `113`, `116`, `124` and `131`.